### DISCUSSION TITLE
Any plan for separated CLI tool?

### DISCUSSION THREAD

**Discussion by suhdonghwi** [williamboman/nvim-lsp-installer#818] [Q&A] [answered]
As far as I know this plugin is the best tool to manage installations across many programming languages. I am sure that the features of this plugin would also be highly useful for developers who are not interested in (neo)vim.

Is there any plan to separate this plugin into "core features CLI tool" and "vim plugin"? (like [fzf](https://github.com/junegunn/fzf) and [fzf.vim](https://github.com/junegunn/fzf.vim))

Thank you.

**Comment by williamboman** [comment, marked as answer]
Hello and thank you! No, there are no plans of building a standalone package manager. There are a couple of reasons why, but the biggest ones are; there's plenty of package managers already and this plugin is very simple in comparison. I'm not sure they're even comparable as this plugin is mostly really just a unified client interface that shells out to other, actual, package managers. Secondly, much of the infrastructure is built on-top of neovim APIs, decoupling this would require quite the effort!

Much of the value proposition I had in mind with this plugin was to embed and streamline more experiences within neovim directly, especially ones that are intrinsically coupled with the editor. VSCode does this via their extension ecosystem, vim have https://github.com/mattn/vim-lsp-settings (and obviously also coc which is great) for example.
